src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+min_read: 5
+date: 2021-03-09
```

Comment:
```suggestion
date: 2021-03-10
```

Had je al een beetje gezien hoe 'commit suggestions' werkt? Daarmee kan je de wijzing direct vanuit Github overnemen (en committen)

Reply:
Nee dat had ik nog niet gezien, maar dat is wel handiger inderdaad.